gensec/customPrecon.py

```diff
@@ -920,162 +1201,185 @@ def model_matrix(bra, atom, builder, damper, thres, logfile=None):
 # Obviously, this is not the most efficient way to do it.  But at least,
```

Comment:
This comment should rather be in a docstring

Reply:
On a second thought: Why do you need these functions? I don't understand them. If they are really needed, a reason should be provided (please let me know what is the reason).